site/src/pages/components/scoped-focusgroup.explainer.mdx

```diff
@@ -763,6 +770,141 @@ Focusgroups have the following additional features:
+4. **Using `no-memory`:** With `focusgroup="toolbar no-memory"`, focus would always go to "Italic" regardless of previous focus
+
+#### Multiple Items with the Entry Priority Attribute (Not Recommended)
+Focusgroup will always prioritize the first element with the `focusgroup-entry-priority` attribute in DOM order when entering a focusgroup segment, or if the items are also being managed by reading-flow, the first such item in reading order, and will not consider any subsequent elements with the same attribute.
+
+```html
+<div focusgroup="toolbar no-memory">
+  <button>Bold</button>
+  <button focusgroup-entry-priority>Italic</button>
+  <button focusgroup-entry-priority>Underline</button>
+  <button>Strikethrough</button>
+</div>
+```
+In this example, "Italic" will receive focus when entering the focusgroup because it appears first in DOM order among elements with `focusgroup-entry-priority`, or if the items are also being managed by reading-flow, it would be the first in reading order among elements with `focusgroup-entry-priority`.
+
+#### Interaction with Nested Focusgroups
+
+When focusgroups are nested, each focusgroup manages its own entry priority independently. The `focusgroup-entry-priority` attribute only affects the focusgroup that directly contains the element.
+
+```html
+<div focusgroup="toolbar" aria-label="Main toolbar">
+  <button>Save</button>
+  <button focusgroup-entry-priority>Print</button>
+
+  <!-- Nested focusgroup with its own entry priority -->
+  <div focusgroup="toolbar" aria-label="Text formatting">
+    <button>Bold</button>
+    <button focusgroup-entry-priority>Italic</button>
+    <button>Underline</button>
+  </div>
+
+  <button>Close</button>
+  <button focusgroup-entry-priority>Exit</button>
+</div>
+```
+
+In this example, there are two distinct focusgroups, and three focusgroup segments:
+
+**Main toolbar**: `[Save, Print]`, `[Close, Exit]`
+- Entry priority: "Print" (has `focusgroup-entry-priority`)
+
+**Nested toolbar**: `[Bold, Italic, Underline]`
+- Entry priority: "Italic" (has `focusgroup-entry-priority`)
+
+**Sequential navigation behavior:**
+1. **Tab into main toolbar:** Focus goes to "Print" (entry priority)
+2. **Tab from "Print":** Focus moves to "Italic" (entry priority in nested toolbar)
+3. **Tab from nested toolbar:** Focus moves to "Close" (continuing in main toolbar)
+4. **Shift+Tab into main toolbar from after:** Focus goes to "Print" (entry priority)
+
+**Directional navigation behavior:**
+- Arrow key navigation within the main toolbar moves between "Save", "Print", "Close", and "Exit", but skips over the nested toolbar
+- Arrow key navigation within the nested toolbar moves between "Bold", "Italic", and "Underline"
+- Each focusgroup maintains independent arrow navigation
+
+This demonstrates how entry priority works independently for each nested focusgroup and how exiting and entering nested focusgroups respects each segment's entry priority.
```

Comment:
If each segment can have it's own entry priority, the above 4. step should focus on "Exit" instead of "Print".